src/index.html

```diff
@@ -2,15 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchBar"
+      action="https://www.w3schools.com/html/html_forms_attributes.asp"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="inpt"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      <img
+        src="images/Search.svg"
+        alt="search icon"
+        class="search-icon"
+      >
+    </form>
+
+    <form
+      class="searchBar"
+      action="https://www.w3schools.com/html/html_forms_attributes.asp"
+      method="get"
+      data-qa="small">
+      <input
+        class="inptSmall"
```

Comment:
```suggestion
        class=" input inpt--small"
```
look checklist point 2